warehouse/upload.go

```diff
@@ -1595,7 +1594,23 @@ func (job *UploadJobT) Aborted(attempts int, startTime time.Time) bool {
 }
 
 func (job *UploadJobT) setUploadError(statusError error, state string) (string, error) {
-	pkgLogger.Errorf("[WH]: Failed during %s stage: %v\n", state, statusError.Error())
+	defer func() {
+		pkgLogger.Errorw("upload error",
+			logfield.JobID, job.upload.ID,
+			logfield.Status, state,
```

Comment:
prefixing with `upload_` to remove ambiguity: 
```suggestion
			logfield.UploadStatus, state,
```